componets/alerts/alert.css

```diff
@@ -0,0 +1,53 @@
+:root {
+  --success-color: #a7f3d0;
+  --success-text-color: #065f46;
+  --info-color: #93c5fd;
+  --info-text-color: #1e40af;
+  --warning-color: #fde68a;
+  --warning-text-color: #92400e;
+  --danger-color: #fca5a5;
+  --danger-text-color: #991b1b;
+}
+
+.alert {
+  width: 30%;
+  padding: 1rem;
+  border-radius: 0.3125rem;
+  margin-bottom: 0.5rem;
```

Comment:
You should use px  for units  less than 1rem

Reply:
ok